crates/hyperqueue/src/client/commands/submit.rs

```diff
@@ -309,7 +386,7 @@ pub async fn submit_computation(
         program: program_def,
         resources,
         pin,
-        priority,
+        priority: priority.unwrap_or(0),
```

Comment:
Maybe we could introduce a separate function that will take the opts and fill in all defaults, so that it is on one place? Or at least we could fill the defaults on a single spot at the beginning of this function.